spec/models/exercise_spec.rb

```diff
@@ -2,11 +2,12 @@
 
 require 'rails_helper'
 require 'stdout_example'
+require 'zip'
 
 RSpec.describe Exercise, type: :model do
-  describe '.create_submission' do
-    subject(:exercise) { FactoryGirl.create(:exercise) }
+  subject(:exercise) { FactoryGirl.create(:exercise) }
 
+  describe '.create_submission' do
```

Comment:
The convention is to prepend instance method names with a hash e.g. `#create_submission`